app/models/user/course_status/next_in_line.rb

```diff
@@ -11,7 +11,7 @@ def next_from_top
   end
 
   def last_section
-    user.quizzes.where(quizable_type: "Section").order(created_at: :desc).first.quizable
+    user.quizzes.where(quizable_type: "Section").order(updated_at: :desc).first
```

Comment:
Isn't this lying? We're expecting a `section` but returning a `quiz`. Any particular reason for the change on removing the `quizable`? We could rename the method to `#last_quiz` if you want to assign the quiz to variable instead of the section.

Reply:
@caioertai the reason for the change was that I needed to get the last quiz in order to calculate the quiz score here:
![Screen Shot 2021-04-06 at 8 08 28 AM](https://user-images.githubusercontent.com/59634117/113724233-61409e00-96af-11eb-8329-a9850f6431ac.png)

I will just make two separate methods, one for last_section and one for last_quiz, so we aren't lying anymore :)